_includes/docs/user-guide/contribution/widgets-development.md

```diff
@@ -119,6 +119,7 @@ Below is brief description of widget context properties:
 | widgetTitle                      | String             | If set, will override configured widget title text. **updateWidgetParams()** function must be called after this property change. |
 | detectChanges()                  | Function           | Trigger change detection for current widget. Must be invoked when widget HTML template bindings should be updated due to widget data changes. |
 | updateWidgetParams()             | Function           | Updates widget with runtime set properties such as **widgetTitle**, **hideTitlePanel**, etc. Must be invoked in order these properties changes take effect. |
+| pageLink(pageSize, page, textSearch, sortOrder) | [PageLink](https://github.com/thingsboard/thingsboard/blob/v3.4/ui-ngx/src/app/shared/models/page/page-link.ts#L96) | Using to create sorting configuration for get requests. **pageSize** - specifies how many entities will be in the page, **page** - specifies what page with entities will be get, **textSearch** - specifies what text must be included in entities, **sortOrder** - specifies in which order entities will be get. |
```

Comment:
Add parameter "release.wd_examples_commit" to "_config.yml". use the constant instead of "13e6b10b7ab830e64d31b99614a9d95a1a25928a" .

See how release.pe_edge_full_ver is used for example.